### ISSUE TITLE
Normalize adds extra height to wide icon since 10.0.5

### ISSUE THREAD

**Issue by thybzi** [nfroidure/svgicons2svgfont#152]
## Issue

I'm a gentledev i:
- [x] fully read the README recently
- [x] searched for existing issues
- [x] checked i'm up to date with the latest version of the project

### Expected behavior
wide glyph height based on content height

### Actual behavior
wide glyph height based on content width (empty vertical space above icon content)

### Steps to reproduce the behavior
1. Use the following source svg:
```xml
<svg width="32" height="12" viewBox="0 0 32 12" xmlns="http://www.w3.org/2000/svg">
    <path d="m1.64861,0l14.35139,9.01l14.35139,-9.01l1.64861,2.42447l-14.62616,9.1825c-0.8347,0.52404 -1.91298,0.52404 -2.74768,0l-14.62616,-9.1825l1.64861,-2.42447z"/>
</svg>
```
</svg>
2. convert with `normalize: true`

Possibly broken in #137, works perfectly up to version 10.0.4, broken in 10.0.5

### Debugging informations
- `node -v` result:
```
v16.14.2
```

- `npm -v` result:
```
8.5.0
```
If the result is lower than 6.9.5, there is
 poor chances i even have a look to it. Please,
 use the last [NodeJS LTS version](https://nodejs.org/en/).

### Use cases

- [ ] I will/have implement the feature

**Comment by georgemincof** [comment]
#153

**Comment by Magiczne** [comment]
@nfroidure the change implemented in https://github.com/nfroidure/svgicons2svgfont/pull/137 was indeed a breaking change as it changes the icon height, not actually preserving aspect ratio.

Will it be fixed?

**Comment by darylteo** [comment]
This issue seems to have been fixed somehow in `12.0.0` as listed here https://www.npmjs.com/package/svgicons2svgfont?activeTab=versions

But this doesn't seem to be an official release?

Forcing this specific module to 10.0.4 or 12.0.0 with npm overrides worksaround this issue.
